### PR TITLE
use init container to copy sshd and openssl command

### PR DESCRIPTION
**Pull request by xudifsd**
More test needed. We can merge it later.

Only tested in `deepscale` container image.

This speed up container init process from 30s to 2s.

### CODE REVIEW

**Comment by xudifsd** [comment]
I have test all images used by dlts users in all clusters. Work as expected except for `indexserveregistry.azurecr.io/deepscale:1.0.post01` and `indexserveregistry.azurecr.io/deepscale:1.0.post1`, they can not be pulled in p40-dev1 cluster, but I think they have same environment as `indexserveregistry.azurecr.io/deepscale:1.0.post0`. So I consider this PR to be good enough.

List of images used by jobs in case you interested:
```
$ for i in `ls ../../..` ; do ./get_images.py -c ../../../$i/src/ClusterBootstrap/config.yaml -i ../../../$i/src/ClusterBootstrap/deploy/clusterID.yml ; done | sort | uniq -c
     36 dlws/dltsinference
     12 dlws/horovod:0.13.10-tf1.12.0-torch1.0.1-py3.5
  37416 indexserveregistry.azurecr.io/deepscale:1.0.post0
     36 indexserveregistry.azurecr.io/deepscale:1.0.post01
     60 indexserveregistry.azurecr.io/deepscale:1.0.post1
     12 indexserveregistry.azurecr.io/deepscale:745
    132 indexserveregistry.azurecr.io/dlts-data-transfer-image
     24 indexserveregistry.azurecr.io/dlts-data-transfer-image:7299912
     12 indexserveregistry.azurecr.io/dlts-data-transfer-image:7537656
    360 indexserveregistry.azurecr.io/dlts-data-transfer-image:latest
     84 indexserveregistry.azurecr.io/test/deepscale:1.1_cuda10.0_nccl2.4.2_cudnn7.6.0.64_mlnx4.6-1.0.1.1_ompi4.0.1_python3_tensorflow1.14.0_pytorch1.1.0_horovod0.16.4
     60 indexserveregistry.azurecr.io/test/dlts-inference:deepscale.1.0.post0
     24 pytorch/pytorch:0.4.1-cuda9-cudnn7-devel
     12 pytorch/pytorch:nightly-devel-cuda10.0-cudnn7
    312 ubuntu
```

**Comment by xudifsd** [comment]
ready for merging

File: src/docker-images/init-container/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+RUN apt-get update && apt-get install -y wget gzip build-essential
+
+WORKDIR /ssh_build
+
+COPY build-openssh-static.sh /ssh_build
+RUN sh build-openssh-static.sh
+
+#FROM python:3.8.0-alpine3.10
+FROM ubuntu:16.04
```
Comment (Anbang-Hu) [inline src/docker-images/init-container/Dockerfile]: Does SSH build in 16.04 work for other versions of Ubuntu?

Comment (xudifsd) [reply src/docker-images/init-container/Dockerfile]: Yes, have it tested in 16.04 and 18.04.

**Pull request merged**